### PR TITLE
Collections UI minor fixes

### PR DESCRIPTION
**Pull request by dvail**
## Description

Two minor UI tweaks:

1. Fixes the misaligned "expandable section icon" in production builds
2. Ensures the Save/Cancel button section of the collection form is always pinned at the bottom of the page

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

Icon before:
image

Icon after:
image

Button toolbar before:
image

Button toolbar after:
image

### CODE REVIEW

**Comment by dvail** [comment]
Current dependencies on/for this PR:
* master
  * **PR #4543** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/4543" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/stackrox/4543?utm_source=stack-comment).

File: ui/apps/platform/src/Containers/Collections/CollectionForm.css [hunk 1]
```diff
@@ -16,4 +16,13 @@
...
   outline: none;
+}
+
+.collection-form-expandable-section .pf-c-expandable-section__toggle-icon {
+  /*
+    Some conflict with TW styles in production build only is causing this element to be
+    misaligned due to an excessive calculated height. Setting this max prevents it from
+    growing too large. This likely can be removed when we fully migrate to PF.
+  */
+  max-height: 2rem;
```
Comment (pedrottimark) [inline ui/apps/platform/src/Containers/Collections/CollectionForm.css:27]: Can you add empty line at the end of the file?

**Review by pedrottimark — APPROVED** [review]
Alignment. Invisible when it is intuitive. Distracting when it is not.

File: ui/apps/platform/src/Containers/Collections/CollectionForm.css (hunk 1 above)
Comment (dvail) [reply ui/apps/platform/src/Containers/Collections/CollectionForm.css:27]: I'll piggyback this on my next PR to avoid an additional CI run.

**Pull request merged**